containers/FlyBox.js

```diff
@@ -1,133 +1,100 @@
-import React, { Component } from 'react';
-import { StyleSheet, Text, View, Button, FlatList } from 'react-native';
-import Fly from '../components/Fly';
-import { setFlies } from '../actions'
+import React, { Component } from "react";
+import { StyleSheet, Text, View, Button, FlatList } from "react-native";
+import Fly from "../components/Fly";
+import { setFlies } from "../actions";
 import { connect } from "react-redux";
-
-// Rachael: this is mocked out data that will be removed once the API requests are up and running
-const TestFlies = [
-    {
-        id: 1,
-        name: "Parachute Adams",
-        size: 18,
-        color: "tan",
-        category: "dry",
-        amount: 2,
-        isFavorite: false,
-        user_id: 1,
-        created_at: "07/18/2020",
-        updated_at: null
-    },
-    {
-        id: 2,
-        name: "Copper John",
-        size: 20,
-        color: "copper",
-        category: "midge",
-        amount: 1,
-        isFavorite: false,
-        user_id: 1,
-        created_at: "07/19/2020",
-        updated_at: null
-    },
-    {
-        id: 1,
-        name: "Zebra Midge",
-        size: 22,
-        color: "red",
-        category: "midge",
-        amount: 3,
-        isFavorite: false,
-        user_id: 1,
-        created_at: "07/17/2020",
-        updated_at: null
-    }
-]
+import { fetchFlies } from "../ApiCalls";
 
 const FlatListItemSeparator = () => {
-    return (
-        <View style={{ height: 1.5, width: "100%", backgroundColor: "#7A5C58"}} />
-    )
-}
+  return (
+    <View style={{ height: 1.5, width: "100%", backgroundColor: "#7A5C58" }} />
+  );
+};
 
 class FlyBox extends Component {
-    constructor(props) {
-        super(props);
-    }
+  constructor(props) {
+    super(props);
+  }
 
-    async componentDidMount() {
-        this.props.setFlies(TestFlies)
-    }
+  async componentDidMount() {
+    fetchFlies().then((data) => this.props.setFlies(data.data));
+  }
```

Comment:
This set up is perfect to store live data from the FlyBox API.  Now I believe we are in a good position to continue other iteration to reach our MVP.

Reply:
Agreed - it was nice and straight forward so should be helpful with the other aspects of our MVP

Reply:
Thank you for getting this done! now the project begins!!